test/extended/dns/dns.go

```diff
@@ -554,4 +557,61 @@ var _ = Describe("[sig-network-edge] DNS", func() {
 		pod := createDNSPod(f.Namespace.Name, cmd)
 		validateDNSResults(f, pod, expect, times)
 	})
+
+	It("should answer queries using the local DNS endpoint", func() {
+		ctx := context.Background()
+
+		By("starting an exec pod for running queries")
+
+		queryPodExec, err := exutil.NewPodExecutor(oc, "dnsquery", image.ShellImage())
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		queryPod, err := f.ClientSet.CoreV1().Pods(oc.Namespace()).Get(ctx, "dnsquery", metav1.GetOptions{})
+		if err != nil {
```

Comment:
Is there a reason not to use `o.Expect(err).NotTo(o.HaveOccurred())` here?

Reply:
Just was copying and pasting code from other tests, but seems reasonable to use.
Done.